examples/troubleshooter/test.js

```diff
@@ -3,7 +3,7 @@ let video = true;
 let audio = true;
 
 let iceServers = [];
-const webRTCTroubleshooter = WebRTCTroubleshooter.default;
```

Comment:
In the `src/index.js` reference that we are `require`ing, I chose not to export the module as default. Babel 6.x treats the `exports` object when `export default ...` as export["default"]. I chose to collectively export all the modules with `export *`.